### PR TITLE
MRO Notebook

### PR DESCRIPTION
**Pull request by acpaquette**
Added Mro isd generation notebook

### CODE REVIEW

File: notebooks/write_MroCtxPds3LabelNaifSpiceDriver.ipynb [hunk 1]
```diff
@@ -0,0 +1,168 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Writing out a USGSCSM ISD from a PDS3 LRO LROC NAC image"
```
Comment (paarongiroux) [inline notebooks/write_MroCtxPds3LabelNaifSpiceDriver.ipynb]: Forgot to change to MRO here

File: notebooks/write_MroCtxPds3LabelNaifSpiceDriver.ipynb [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    "\n",
+    "The default config file is located at `ale/config.yml` and is copied into your home directory at `.ale/config.yml` on first use of the library. The config file can be modified using a text editor. `ale.config` is loaded into memory as a dictionary. It is used to find metakernels for different missions. For example, there is an entry for MRO that points to `/usgs/cpkgs/isis3/data/mro/kernels/mk/` by default. If you want to use your own metakernels, you will need to udpate this path. For example, if the metakernels are located in `/data/ctxmro/mk/` the MRO entry should be updated with this path. If you are using the default metakernels, then you do not need to update the path.\n",
```
Comment (kaitlyndlee) [inline notebooks/write_MroCtxPds3LabelNaifSpiceDriver.ipynb]: There is a typo that probably exists in the other notebook as well. "If you want to use your own metakernels, you will need to udpate this path." Udpate should be update.

File: notebooks/write_MroCtxPds3LabelNaifSpiceDriver.ipynb [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+    }
+   ],
+   "source": [
+    "# printing config displays the yaml formatted string\n",
+    "print(ale.config)\n",
+    "\n",
+    "# config object is a dictionary so it has the same access patterns\n",
+    "print('MRO spice directory:', ale.config['mro'])\n",
+    "\n",
+    "# updating config for new LRO path in this notebook \n",
```
Comment (paarongiroux) [inline notebooks/write_MroCtxPds3LabelNaifSpiceDriver.ipynb]: change to MRO

Comment (thareUSGS) [reply notebooks/write_MroCtxPds3LabelNaifSpiceDriver.ipynb]: Looks good

**Pull request merged**